composer.json

```diff
@@ -20,6 +20,7 @@
         "doctrine/common": "~2.4",
         "twig/twig": "~1.28|~2.0",
         "psr/cache": "~1.0",
+        "psr/container": "^1.0@dev",
```

Comment:
Just reminder comment: Remove `@dev` annotation when the package will be stable.